### DISCUSSION TITLE
Misurazione, accensione e spegnimento di Lavatrice/Asciugatrice

### DISCUSSION THREAD

**Discussion by barbatrukko** [indomus/forum#859] [HUB: Home Assistant]
Salve,
sto riorganizzando il bagno di servizio e sto cercando di mettere insieme affiancate Asciugatrice e Lavatrice che prima erano in due ambienti diversi.
Prima di metterle insieme ognuna delle due era gestita/monitorata da una presa shuko Meross.
Ora dovendo metterle vicine e con poco spazio in larghezza e profondità per le prese, avevo pensato ad una 504 con 2 shuko e gestire a monte il controllo.
Servirebbe dunque qualcosa da chiudere in una scatola/cassettina a monte che abbia 2 canali di misurazione e di accensione/spegnimento e che regga una potenza fino a 3.3 KW (anzi mi pare che i nuovi contatori  arrivino a 4KW)
Quale componente mi consigliate facilmente integrabile in Home Assistant?
Devo misurare e staccare e attaccare all'occorrenza i due canali, magari su input di carico eccessivo.

**Comment by essegi65** [comment]
Per gestire a monte il controllo potresti usare tipo dei sonoff pow2 da mettere in una scatola di derivazione da dove partono le linee che arrivano a i tuoi elettrodomestici. (i sonoff pow non sono adatti per le scatole 504). Non mi sembra esistano circuiti a 2 canali che possono sopportare i 16A di una lavatrice / essiccatore. Io avendoli in una stanza lavanderia ho optato per un quadretto da esterno con tutto (sonoff, relè, prese, ecc.) all'interno.

**Reply by barbatrukko** [reply]
Questo potrei farlo, perché il problema è doppio:
da una parte devo mettere i POW  o equivalenti, dall'altra devo trovare un modo per collegare le shuko in un posto dove c'è comunque rischio acqua. La fregatura è che tutti i produttori non ti consentono di tagliare il cavo di alimentazione pena caduta garanzia, altrimenti mi portavo direttamente i cavi dove mi faceva più comodo.
Ho già delle scatole di derivazione in zona e volendo potrei metterne una nuova da dedicare a quella zona con un occhio a espansioni future.
